src/CardSubtitle.js

```diff
@@ -0,0 +1,34 @@
+import classNames from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+import elementType from 'prop-types-extra/lib/elementType';
+
+import { createBootstrapComponent } from './ThemeProvider';
+
+class CardSubtitle extends React.Component {
+  static propTypes = {
+    /**
+     * @default 'card-subtitle'
+     */
+    bsPrefix: PropTypes.string,
+
+    componentClass: elementType
+  };
+
+  static defaultProps = {
+    componentClass: 'h6'
```

Comment:
generally we don't want to default anything to a `h*` tag because the one you use is completely context specific, e.g how deep in the app it is etc. What we might want to do is default to a `div className='h6'/>` instead that a user can override as via the componentClass

Reply:
This makes sense, will change the Title/Subtitle to that.